app/assets/javascripts/application.js

```diff
@@ -45,34 +44,34 @@
  */
 window.updateLinks = function (loc, zoom, layers, object) {
   $(".geolink").each(function (index, link) {
-    var href = link.href.split(/[?#]/)[0],
-        args = Qs.parse(link.search.substring(1)),
-        editlink = $(link).hasClass("editlink");
+    let href = link.href.split(/[?#]/)[0];
```

Comment:
Not sure if this is a good and correct change:
```suggestion
    let href = link.protocol + '//' + link.hostname + link.pathname;
```

Reply:
I'm not actually changing that line though? Other than replacing `var` with `let`?

Reply:
I thought the general goal was to replace complex code with modern APIs.
If this is an external library or own code is for me not important. 
Do what you want in this PR.